app/products/calls/screens/host_controls/host_controls.tsx

```diff
@@ -125,6 +157,20 @@ export const HostControls = ({
                     onPress={makeHostPress}
                     text={makeHostText}
                 />
+                <SlideUpPanelItem
+                    leftIcon={'account-outline'}
+                    leftIconStyles={styles.iconStyle}
+                    onPress={profilePress}
+                    text={profileText}
+                />
+                <SettingSeparator lineStyles={styles.separator}/>
+                <SlideUpPanelItem
+                    leftIcon={'minus-circle-outline'}
+                    leftIconStyles={[styles.iconStyle, styles.red]}
+                    onPress={removePress}
+                    text={removeText}
+                    textStyles={styles.red}
```

Comment:
this style is not needed, you could just set the prop for `destructive={true}`

Reply:
Awesome, thanks.